server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/cucumber/SetMailboxesMethodStepdefs.java

```diff
@@ -130,9 +130,20 @@ public void shareMailbox(String owner, String mailboxName, String shareTo, Strin
         shareMailboxWithRight(owner, mailboxName, rights, shareTo);
     }
 
-    @When("^renaming mailbox \"([^\"]*)\" to \"([^\"]*)\"")
-    public void renamingMailbox(String actualMailboxName, String newMailboxName) throws Throwable {
-        Mailbox mailbox = mainStepdefs.mailboxProbe.getMailbox("#private", userStepdefs.getConnectedUser(), actualMailboxName);
+    @When("^\"([^\"]*)\" renames the mailbox, owned by \"([^\"]*)\", \"([^\"]*)\" to \"([^\"]*)\"$")
+    public void renamingMailbox(String user, String mailboxOwner, String actualMailboxName, String newMailboxName) throws Throwable {
+        Mailbox mailbox = mainStepdefs.mailboxProbe.getMailbox("#private", mailboxOwner, actualMailboxName);
+        userStepdefs.connectUser(user);
```

Comment:
does this method change the active user ?

Reply:
Yes, but I don't want to change the whole cucumber tests know....